src/main/java/com/musala/atmosphere/agent/DeviceChangeHandler.java

```
@@ -144,7 +130,7 @@ public void run() {
             if (publishId != null && !publishId.isEmpty()) {
                 try {
                     onDeviceListChanged(publishId, false /* device disconnected */);
-                } catch (CommandFailedException | NotBoundException e) {
+                } catch (CommandFailedException e) {
 
```

log error here, do not just silently ignore it